tests/e2e/specs/psi-activation.test.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import { switchUserToAdmin, visitAdminPage, activatePlugin, deactivatePlugin } from '@wordpress/e2e-test-utils';
+import { resetSiteKit } from '../utils/reset';
+
+describe( 'PageSpeed Insights Activation', () => {
+	beforeEach( async() => {
+	} );
+
+	afterEach( async() => {
+	} );
+
+	it( 'Dashboard should display Activate PageSpeed Insights CTA', async() => {
+
+		await resetSiteKit();
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-dashboard' );
+		await page.waitForSelector( '.googlesitekit-cta__title' );
+		const welcomeTitle = await page.$x(
```

Comment:
As mentioned in another PR, #330, the assertions offered by `expect-puppeteer` make these kind of tests way more readable than XPath.

You could write:

```js
await expect( page ).toMatchElement( '.googlesitekit-cta__title', { text: 'Activate PageSpeed Insights.' } );
```